src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,425 @@
+| ChunkTransformationMatrixAxis | REQUIRED if `ChunkTransformationMatrix` is present  | [arrays][] of [strings][]              | Describe the axis of the ChunkTransformationMatrix (for examples: `["X", "Y"]` or `["Z", "Y", "X"]`)                                                                                                                                                                                                                                                                                                                                                                                   |
+
+An example of chunk transformations JSON metadata for `chunk-01` and `chunk-05` of Figure 2
+is shown below:
+
+![Figure 2](images/microscopy_chunks2.png "Example figure for chunks transformations")
+
+Figure 2: Example figure for chunks transformations.
+
+In this example, there is no scaling and `chunk-01` is at the origin.
+`chunk-05` is translated by 5 um along `X+` and by 3 um along `Y+`.
+
+**`*_chunk-01_<modality_suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 0],
+                                      [0, 1, 0],
+                                      [0, 0, 1]],
+        "ChunkTransformationMatrixAxis": ["X", "Y"]
+}
+```
+
+**`*_chunk-05_<modality_suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 5],
+                                      [0, 1, 3],
+                                      [0, 0, 1]],
+        "ChunkTransformationMatrixAxis": ["X", "Y"]
+}
+```
+
+#### Example of sidecar JSON file (`*_<modality_suffix>.json`)
+```JSON
+{
+        "Manufacturer": "Hamamatsu",
+        "ManufacturersModelName": "C9600-12",
+        "PixelSize": [0.23, 0.23],
+        "PixelSizeUnits": "um",
+        "Magnification": 40,
+        "BodyPart": "BRAIN",
+        "BodyPartDetails": "corpus callosum",
+        "SampleEnvironment": "exvivo",
+        "SampleFixation": "4% paraformaldehyde, 2% glutaraldehyde",
+        "SampleStaining": "LFB",
+        "SliceThickness": 5,
+        "TissueDeformationScaling": 97
+}
+```
+
+## Required Samples file (`samples.tsv`)
+
+For Microscopy data, the [Samples file](../03-modality-agnostic-files.md#samples-file) is REQUIRED.
+
+Additional optional columns in `samples.tsv` MAY be used to describe other samples' attributes.
+
+## Recommended participant data (`participants.tsv`)
+
+For Microscopy data, we RECOMMEND to make use of the columns `species`, `strain` and `strain_rrid`
```

Comment:
I suggest to write "As for other datatypes in BIDS, the `participants.tsv` file is required and MUST contain the column `participant_id`", and then elaborate on the optional columns.

Reply:
I agree, thank you, I integrated your suggestion with a slight modification as "participants.tsv" is recommended in BIDS and not required.